sunpy/visualization/animator/base.py

```diff
@@ -571,6 +588,26 @@ def update_plot(self):
         """
         Abstract method for updating plot.
 
-        Must exists here but be defined in subclass.
+        Must exist here but be defined in subclass.
 
         """
+
+
+def edges_to_centers_nd(axis_range, edges_axis):
```

Comment:
Is there a test for this?

Reply:
No.  But as it's a new function there could be.